internal/cmd/commands/database/init.go

```diff
@@ -37,17 +51,7 @@ type InitCommand struct {
 	// deferred function on the Run method.
 	configWrapperCleanupFunc func() error
 
-	flagConfig                                   []string
-	flagConfigKms                                string
-	flagLogLevel                                 string
-	flagLogFormat                                string
-	flagMigrationUrl                             string
-	flagSkipInitialLoginRoleCreation             bool
-	flagSkipInitialAuthenticatedUserRoleCreation bool
-	flagSkipAuthMethodCreation                   bool
-	flagSkipScopesCreation                       bool
-	flagSkipHostResourcesCreation                bool
-	flagSkipTargetCreation                       bool
+	InitFlags InitFlags
```

Comment:
Could we add some docs to this field? We have to make it clear that new flags should be added to the struct, not as a new field. Also, should this field be exported?